### PR TITLE
add infos and validations to users

### PR DESCRIPTION
**Pull request by zuccamia**

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -3,4 +3,9 @@ class User < ApplicationRecord
   # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
+
+  validates :first_name, presence: true, length: { maximum: 30 }
```
Comment (worksofgenius) [inline app/models/user.rb]: Maybe minimum is better?

File: db/migrate/20210123080106_add_info_to_users.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
+class AddInfoToUsers < ActiveRecord::Migration[6.1]
+  def change
+    add_column :users, :first_name, :string
+    add_column :users, :last_name, :string
+    add_column :users, :phone_number, :string
+    add_column :users, :languages, :string, array: true
+    add_column :users, :age_range, :string
```
Comment (worksofgenius) [inline db/migrate/20210123080106_add_info_to_users.rb:7]: Age_range should be inclusion.

**Review by worksofgenius — COMMENTED** [review]
Please see comments.

File: app/models/user.rb [hunk 3]
```diff
@@ -4,8 +4,10 @@ class User < ApplicationRecord
   devise :database_authenticatable, :registerable,
          :recoverable, :rememberable, :validatable
 
-  validates :first_name, presence: true, length: { maximum: 30 }
-  validates :last_name, presence: true, length: { maximum: 30 }
+  AGE_RANGE = %w( 18-29 30-29 40-49 50-59 60-69 70+ ).freeze
```
Comment (worksofgenius) [inline app/models/user.rb]: Please change to 30-39.

Comment (zuccamia) [reply app/models/user.rb]: Thanks for the catch! Please review updated commit.

**Pull request merged**